spec/models/instance_spec.rb

```diff
@@ -1,6 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe Instance, type: :model do
+  let(:instance) { build(:instance) }
+
+  context 'when host format is invalid' do
+    it "is invalid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.